aws-datastore/src/androidTest/java/com/amplifyframework/datastore/AppSyncClientInstrumentationTest.java

```diff
@@ -303,7 +304,7 @@ public void testAllOperations() throws DataStoreException {
     private <T> T awaitResponseData(
             Await.ResultErrorEmitter<GraphQLResponse<T>, DataStoreException> resultErrorEmitter)
             throws DataStoreException {
-        final GraphQLResponse<T> response = Await.result(resultErrorEmitter);
+        final GraphQLResponse<T> response = Await.result(TimeUnit.SECONDS.toMillis(20), resultErrorEmitter);
```

Comment:
😳

Reply:
This change increases it from the default of 5 seconds to 20 seconds, and it actually is the difference in getting the tests to pass on CircleCI!   It passes every time locally with a 5 second timeout, but for some reason takes longer on CircleCI.